core/domain/exp_services.py

```diff
@@ -941,14 +1190,32 @@ def save_exploration_summary(exp_summary):
 
 
 def delete_exploration_summary(exploration_id):
-    """Delete an exploration summary model."""
+    """Delete an exploration summary model.
+
+    Args:
+        exploration_id: str.The exploration id of the exploration 
+        domain object to be deleted.
+    """
 
     exp_models.ExpSummaryModel.get(exploration_id).delete()
 
 
 def revert_exploration(
         committer_id, exploration_id, current_version, revert_to_version):
-    """Reverts an exploration to the given version number. Commits changes."""
+    """Reverts an exploration to the given version number. Commits changes.
+    Args:
+        committer_id: str.id of the person who made the commit.
+        exploration_id: str.id of the exploration domain object to be 
+                        reverted to the cuurent version.
+        current_version: float.The present version.
+        revert_to_version: float.The last version.
+
+    Raises:
+        Exception: When current_version is greater than exploration_model's 
```

Comment:
Exception: The given version does not match the version of the currently-stored exploration model.